src/extension/background-script/actions/ln/makeInvoice.js

```diff
@@ -3,7 +3,7 @@ import state from "../../state";
 
 const makeInvoice = async (message, sender) => {
   if (message.args.memo === undefined) {
-    message.args.memo = "Alby invoice memo";
+    message.args.memo = memo.args.defaultMemo || "Alby invoice memo";
```

Comment:
maybe instead of the if and the line 11 we do something like: 

```
const memo = message.args.memo || message.args.defaultMemo || "Alby invoice";
```